About/About.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="About_style.css">
+    <script src="About.js"></script>
+</head>
+
+<body>
+    <!-- Hambergur menu -->
+    <div id="menu">
+        <div id="menu_items">
+            <div class="close"><button class="exit">X</button></div>
+            <div class="menu_item">
+                <a href="/Home/Home.html">Home</a>
+            </div>
+            <div class="menu_item">
+                <a href="/About/About.html">About</a>
+            </div>
+            <div class="menu_item">
+                <a href="#">Contact</a>
+            </div>
+            <div class="menu_item">
+                <a href="#">Login</a>
+            </div>
+        </div>
+    </div>
+    <div class="titlebar">
+        <div class="elements">
+            <img src="/images/Rectangle.png" alt="">
+            <img src="/images/twitter.png" alt="">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </div>
+    </div>
+    <!-- Nav bar -->
+    <nav id="nav_bar">
+        <button class="mob_menu" id="mobmenu"><img src="/images/mobile_menu 1.png" alt=""></button>
+        <div id="logo">
+            <a href="/Home/Home.html" class="cool-button">
+                <h1 class="cool-heading">GamerZportal</h1>
+            </a>
+            <div class="nav_elements">
+                <a href="/About/About.html">About</a>
+                <a href="#">News</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+            </div>
+        </div>
+    </nav>
+    <!-- main body -->
+    <section class="mainbody">
+        <div class="firstbod">
+            <div class="title">
+                <p id="welcome">
+                    "Welcome to GamerZportal"
+                </p>
+                <p id="desc_2">
+                    Game Forum 2023
+                </p>
+            </div>
+            <div class="description">
+                <p>
+                    Welcome to the Gaming Forum, the ultimate hub for gamers, game developers, and enthusiasts from
+                    around
+                    the
+                    world! Our virtual platform serves as a gathering place where experts, academics, and passionate
+                    gamers
+                    come
+                    together to discuss, explore, and celebrate the fascinating world of gaming.
+                </p>
+            </div>
+            <div class="contact">
+                <div class="part1">
+                    Please contact us per Email for any further questions about Gaming Forum 2023!
+                </div>
+                <div class="email">rdzaman20000@gmail.com</div>
+            </div>
+        </div>
+
+        <section class="part2">
```

Comment:
- [ ] This section of your about me is not properly structured on mobile view, there are overlapping text and the logo as shown in the image below cannot be properly viewed as it is cut off. Please kind make the necessary adjustments by matching your design with the provided template design for the about page 👍 
- your design
![ab-m](https://github.com/Rdzaman20000/GamersPortal/assets/89055502/166eb7a5-244f-4f90-b272-afef443dd6f6)